src/app/modules/budget/budget-edit/currency-codes.ts

```diff
@@ -0,0 +1,181 @@
+export const currencyCodes = [
+  'AED',
```

Comment:
serio nie ma żadnej paczki ani w angularze nie ma wbitych currencycode'ów? może da się wyciągnąć bo jakoś działa currency pipe i potrafi ogarnąć że taki skrót to taki symbol

Reply:
currencyPipe wykorzystuje LOCALE_DATA, do którego nie ma dostępu, można jedynie zwrócić dane dla wybranego locale a my byśmy potrzebowali wszystkie:
https://github.com/angular/angular/blob/540c29cd6bfc4124a30f5f8e7b599d326baf6af4/packages/core/src/i18n/locale_data_api.ts#L100

 Są paczki na npm i zastanawiałem się też nad tym i w sumie nie wiem. Bo z jednej strony nie jestem fanem dodawania niepotrzebnie kolejnych paczek jeśli potrzebuje bardzo mały fragment bo bez sensu rośnie rozmiar strony. My nie potrzebujemy nic poza tablicą kodów, a większość paczek ma dużo dodatkowych pierdół i nwm czy są potrzebne